lta/picker.py

```diff
@@ -78,7 +84,7 @@ def __init__(self, config: Dict[str, str], logger: Logger) -> None:
         self.last_work_end_timestamp = timestamp
         self.lta_ok = False
         self.logger.info(f"Picker '{self.picker_name}' is configured:")
-        for name in EXPECTED_CONFIG:
+        for name in EXPECTED_CONFIG.keys():
```

Comment:
Why do you call `.keys()`?  it should work bare.

Reply:
Ahhh, I did not realize that. I thought moving from List to Dict meant giving up nice things. Python iterators to the rescue.